src/transformers/models/bert/tokenization_bert.py

```diff
@@ -417,7 +431,9 @@ def tokenize(self, text, never_split=None):
         # words in the English Wikipedia.).
         if self.tokenize_chinese_chars:
             text = self._tokenize_chinese_chars(text)
-        orig_tokens = whitespace_tokenize(text)
+        # prevents treating the same character with different unicode codepoints as different characters
+        unicode_normalized_text = unicodedata.normalize("NFC", text)
```

Comment:
As the comment mentions, done for [this part](https://github.com/huggingface/transformers/blob/ae5fc2db87f2fb715c137c4823647f5b24c7f881/tests/models/clip/test_tokenization_clip.py#L90-L94) of the test